core/kubernetes/k8s.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+if [ "$1" == "up" ]; then
+    kubectl apply -f ./openhim
+    kubectl apply -f ./hapi-fhir
```

Comment:
We could point to these resource from the kustomization file then just use `kubectl apply -k .`